pages/templates/pages/candidate_endorsement_index_page.html

```diff
@@ -55,20 +55,54 @@ <h3 class="pull-left mb0">
       <div id="candidates-list">
         <div class="no-result text-center"><h3>No Matching Candidates Found</h3></div>
         <ul class="card-list list">
-          {% for candidate_page in candidates %}
-              {% with candidate_page.candidateendorsementpage.candidate as candidate %}
-                  <li class="card clearfix">
-                      <a href="{% pageurl candidate_page %}">
-                          <div class="card__photo" style="background-image: url('{{ candidate.photo.url }}')"></div>
-                          <div class="card__content">
-                              <span class="card__item name">{{ candidate.name }}</span>
-                              <span class="card__item bold smaller state">{{ candidate.state }} {{ candidate.district }}</span>
-                              <span class="card__item bottom">{% if candidate.primary_date %}<div class="card__item smaller bold">Election: <span class="date hidden">{{ candidate.primary_date|date:"U" }}</span>{{ candidate.primary_date }}</div> {% endif %}{{ candidate.office }}</span>
-                          </div>
-                      </a>
-                  </li>
-              {% endwith %}
+
+          {% for candidate in candidates %}
+          {% with candidateendorsementpage=candidate.candidateendorsementpage %}
```

Comment:
Since we don't have established style guidelines for coding Django templates there is no action needed here, but shouldn't this be indented? It would make the compiled source code uglier but would make the template more readable. Curious to hear your thoughts here.

Reply:
I like when the IDE does the indenting standard for me, and in this case the IDE says this is correct. I'm not sure how to verify the correct standard here since like you said we don't have official guidelines.

Reply:
I actually read a bit about this and most IDE's don't care because there aren't actual official style guides here like there are for regular Python with PEP 8. Just something to think about and maybe standardize in future.